src/EnergyPlus/UnitVentilator.hh

```diff
@@ -67,127 +67,143 @@ struct EnergyPlusData;
 
 namespace UnitVentilator {
 
+    enum class CoilOption
+    {
+        Invalid = -1,
+        None,
+        Both,
+        Heating,
+        Cooling,
+        Num
+    };
+    enum class HeatCoilType
+    {
+        Invalid = -1,
+        Electric,
+        Gas,
+        Water,
+        Steam,
+        Num
+    };
+    enum class CoolCoilType
+    {
+        Invalid = -1,
+        Water,
+        Detailed,
+        HXAssisted,
+        Num
+    };
+    enum class OAControl
+    {
+        Invalid = -1,
+        VariablePercent,
+        FixedTemperature,
+        FixedOA,
+        Num
+    };
+
     struct UnitVentilatorData
     {
-        // Members
-        // Input data
         std::string Name;      // name of unit
         std::string SchedName; // availability schedule
-        int SchedPtr;          // index to schedule
-        int AirInNode;         // inlet air node number
-        int AirOutNode;        // outlet air node number
-        int FanOutletNode;     // outlet node number for fan exit
+        int SchedPtr = 0;      // index to schedule
+        int AirInNode = 0;     // inlet air node number
+        int AirOutNode = 0;    // outlet air node number
+        int FanOutletNode = 0; // outlet node number for fan exit
         // (assumes fan is upstream of heating coil)
-        int FanType_Num;     // Fan type number (see DataHVACGlobals)
+        int FanType_Num = 0; // Fan type number (see DataHVACGlobals)
         std::string FanType; // type of fan
```

Comment:
Similarly should not need to store `FanType` string if you are immediately converting it to an `enum`.